test/test.js

```diff
@@ -0,0 +1,52 @@
+const {createS3UploadParams, reverse} = require("../index");
```

Comment:
Codacy has a fix for the issue: [Replace `createS3UploadParams,·reverse` with `·createS3UploadParams,·reverse·`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
const { createS3UploadParams, reverse } = require("../index");
```